api_tests/lib_sdk/actors/actor.ts

```diff
@@ -159,28 +160,73 @@ export class Actor {
 
     public async accept() {
         if (!this.swap) {
-            throw new Error("Cannot accept inexistent swap");
+            throw new Error("Cannot accept non existing swap");
         }
 
         await this.swap.accept(Actor.defaultActionConfig);
     }
 
     public async fund() {
         if (!this.swap) {
-            throw new Error("Cannot fund inexistent swap");
+            throw new Error("Cannot fund non existing swap");
         }
 
         this.logger.debug("Funding as part of swap @ %s", this.swap.self);
         await this.swap.fund(Actor.defaultActionConfig);
+
+        const entity = await this.swap.fetchDetails();
+        switch (entity.properties.role) {
+            case "Alice":
```

Comment:
This is brittle because it breaks when someone writes "alice" (lowercase)?  Perhaps we could have const here to prevent that.

Reply:
Yes and no :)
entity.properties.role is defined as ` "Alice" | "Bob" `:

https://github.com/comit-network/comit-js-sdk/blob/9d315dc774f8bad58dfd6deba0df98e58fc31219/src/cnd.ts#L105

Meaning, typescript should be able to catch this.

Reply:
oh, cool.